platforms/atlys.py

```diff
@@ -441,6 +441,23 @@
 #            Subsignal("tck", Pins("A15")),
 #        ),
 
```

Comment:
**Don't** think we should block merging on this, but I think we need to figure out how to the VHDCI as a connector.

See https://github.com/m-labs/migen/blob/legacy/mibuild/platforms/papilio_pro.py#L45 and https://github.com/m-labs/migen/blob/11f1b134a9165513d32258cda2bae1e1d590a020/migen/build/platforms/pipistrello.py#L116

Dunno how you "use" a connector in migen/misoc though... @enjoy-digital Do you know how?